spec/rails_helper.rb

```diff
@@ -24,4 +24,13 @@
   config.after(:each) do
     DatabaseCleaner.clean
   end
+
+  def create_category
+    let!(:category) { Category.create(name: 'Lenses') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.